specification/dartLangSpec.tex

```diff
@@ -14545,7 +14545,8 @@ \subsubsection{Generic Method Instantiation}
 Let $i$ be a property extraction expression of the form
 \code{$e$?.\id}, \code{$e$.\id}, or \code{\SUPER.\id}
 (\ref{propertyExtraction}, \ref{superGetterAccessAndMethodClosurization}),
-which is statically resolved to denote an instance method named \id,
+which is statically resolved to denote an instance method named \id{}
+that is not the \CALL{} method of a function type,
```

Comment:
So this simply says that if it's the instance `call` method of a function type, then we don't do instantiation (which will make it a type error), right?

Reply:
Exactly.